File: C# I/03.Operators-And-Expressions/08.PrimeCheck/PrimeCheck.cs
using System;
class PrimeCheck
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        if (number == 1)
        {
            Console.WriteLine("false");
        }

        else if (number <= 0 || number >= 100)
        {
            Console.WriteLine("false");
        }

        else if (number > 0)
        {
            for (int count = 2; count < 10; count++)
            {
                if (number % count == 0 && number != count && number != 2)
                {
                    Console.WriteLine("false");
                    return;
                }
            }
            Console.WriteLine("true");
            return;
        }
        
    }
}

File: JavaScript Fundamentals/01.Operators-And-Expressions/PrimeCheck.js
function solve(args) {
    let number = Number(args[0]);
    for (let i = 2; i < 10; i += 1) {
        if(i === number)
        {
            continue;
        }
        if (number < 0 || number % i === 0) {
            console.log("false");
            break;
        }
        if (number > 0 && i === 9) {
            console.log("true");
            break;
        }
    }
}
solve(['3']);